src/fastoad_cs25/models/weight/mass_breakdown/mass_breakdown.py

```diff
@@ -16,18 +16,23 @@
 import openmdao.api as om
 from fastoad.module_management.service_registry import RegisterSubmodel
 
+from ...constants import PAYLOAD_FROM_NPAX
+from ..constants import SERVICE_MASS_BREAKDOWN
 from .constants import (
     SERVICE_AIRFRAME_MASS,
     SERVICE_CREW_MASS,
     SERVICE_FURNITURE_MASS,
+    SERVICE_MLW_MZFW,
     SERVICE_OWE,
     SERVICE_PAYLOAD_MASS,
     SERVICE_PROPULSION_MASS,
     SERVICE_SYSTEMS_MASS,
 )
-from .update_mlw_and_mzfw import UpdateMLWandMZFW
-from ..constants import SERVICE_MASS_BREAKDOWN
-from ...constants import PAYLOAD_FROM_NPAX
+
+# Define default submodel for MLW evaluation
+RegisterSubmodel.active_models[SERVICE_MLW_MZFW] = (
+    "fastoad.submodel.weight.mass.mlw_mzfw.legacy"  # This avoid breaking legacy code
+)
```

Comment:
As there will be only one submodel registered, these lines are optional. To keep consistency with the other submodels (none are registered as active models) I would remove them.

Reply:
You have a good point here! Thank you for pointing it out.